2018-12-10 23:02:58,808 - logs/poolminer.logs - WARNING - logs/poolminer.logs
2018-12-10 23:02:59,051 - logs/poolminer.logs - WARNING - load_data:{'start': True, 'end': True, 'min_date': None, 'max_date': None}
2018-12-10 23:02:59,055 - logs/poolminer.logs - ERROR - poolminer:
Traceback (most recent call last):
  File "/home/andre/aion/data_science/bokeh/aion_analytics/scripts/dashboards/streaming/poolminer.py", line 186, in poolminer_tab
    all_plot = renderer.get_plot(dmap_all)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/plotting/bokeh/renderer.py", line 162, in get_plot
    plot = super(BokehRenderer, self_or_cls).get_plot(obj, renderer, **kwargs)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/plotting/renderer.py", line 182, in get_plot
    initialize_dynamic(obj)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/plotting/util.py", line 244, in initialize_dynamic
    dmap[dmap._initial_key()]
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/core/spaces.py", line 1132, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/core/spaces.py", line 911, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/core/spaces.py", line 574, in __call__
    ret = self.callable(*args, **kwargs)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/util/__init__.py", line 437, in dynamic_operation
    return self._process(map_obj[key], key)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/core/spaces.py", line 1132, in __getitem__
    val = self._execute_callback(*tuple_key)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/core/spaces.py", line 911, in _execute_callback
    retval = self.callback(*args, **kwargs)
  File "/home/andre/anaconda3/envs/bokeh_aion_analytics/lib/python3.5/site-packages/holoviews/core/spaces.py", line 574, in __call__
    ret = self.callable(*args, **kwargs)
  File "/home/andre/aion/data_science/bokeh/aion_analytics/scripts/dashboards/streaming/poolminer.py", line 74, in load_data
    logger.warning('load_data head:%s',self.df.head())
AttributeError: 'NoneType' object has no attribute 'head'
